crates/sui-benchmark/src/lib.rs

```diff
@@ -518,7 +519,10 @@ impl ValidatorProxy for LocalValidatorAggregatorProxy {
 
 pub struct FullNodeProxy {
     sui_client: SuiClient,
+
+    // Committee and protocol config are initialized on startup and not updated on epoch changes.
     committee: Arc<Committee>,
```

Comment:
What makes it so that that tests don't fail on epoch change if the committee is not updated with a new epoch?